### DISCUSSION TITLE
Test results, Usability with 10 users

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2902] [General Questions & Discussion] [answered]
Here are the results of our trial of Hubs Personal on AWS. "Alessandro" was the host for the event. The objective was to evaluate how easy it is to enter and understand Hubs the first time. The overall result is excellent. Mozilla team, 83.3% is a very high mark in terms of a heuristic analysis. 

[https://drive.google.com/drive/u/0/my-drive](url)

**Collaborative Experience = Answers/(Number of Questions X 100)** 

750/900 = 83.3% 

**Questions once completed the experience**

The questions require a numeric answer. The numeric value gives an indication of how much you agree/disagree with any given point. 

1. How long did it take you to go from the feeling of a technological "wow" to actually feeling like Alessandro is talking and you are with him somewhere:

0 = The whole time it felt like a video game
100 = I felt like Alessandro was with me right away and my mind focused on the surroundings and his talk
Answer: 80

2. Could you hear Alessandro clearly throughout the experience?
0 = Had no audio
100 = Like he was next to me
Answer: 90

3. Were the buttons, labels and texts on the screen easy to understand or did you not really understand the controls?
0 = Did not understand the screen
100 = Very easy to grasp
Answer: 90

4. Was navigation in VR easy or hard? Could you move around intuitively or was it confusing?
0= I was totally lost
100 = very easy and intuitive
Answer: 100

5. Did it make sense to use videos in the 3d environment? 
0 = Totally unnecessary
100 = Very useful
Answer: 100

6. How was the experience in the Virtual trade show?
0 = Boring
100 = Interesting
Answer: 70

7. How was the experience in the Photo studio?
0 = Boring
100 = Interesting
Answer: 80

8. How was the experience in the Photo gallery?
0 = Boring
100 = Interesting
Answer: 60

9. How was the experience in the Outdoor VR seminar?
0 = Boring
100 = Interesting
Answer: 80

**Comment by emclaren** [comment, marked as answer]
This is great @ocularvr ! I'd love to know more about the virtual trade show you created!

Also - the link you shared doesn't seem to work. I think it might link to your own drive - any chance you could re-share this link?

**Reply by ocularvr** [reply]
Hi there,

Sorry for the link, here it is:

https://drive.google.com/file/d/1Uei9VXd3Xk1inbgzKE7TF_xJ9ZOvqb6e/view?usp=sharing

We are making a serious hubs server for a dental photography group who has over 1200 members. We want to share all of our findings with you because we believe that Open Source is the way and I personally grew up with Netscape Navigator so there is an emotional bond here :)

We will run another larger test in mid September and would gladly get your input on any user feedback in particular.

**Reply by emclaren** [reply]
Thanks again for sharing this - like on the other discussion we were having I am going to mark this as an answer to help with filtering in our github discussions for unanswered questions (this beta version of Github Discussions doesn't allow us to mark some discussions as "not in need of an answer", this feature is coming soon though).

**Reply by ocularvr** [reply]
Dear Elgin,
Thank you for such positive feedback. We are really excited about using hubs in the real world. My task is to push this development into the real world and make non IT people get onboard and really use this amazing tool. Its really easy for clients to think of the next level of development, you have done so much that it should be up to people like me to convince the clients that the key to success is actual using the tool to get to their end. 

The inspiration for my understanding of hubs comes from ideas from people like Alan Cooper talking about "The inmates are running the asylum"

We want to avoid over complicating the IT and keeping the interface "invisible" to the end user. This end should be as in the case of my clients, hosting events and meetings and not messing around with code and software. Im in love with Hubs 👍 

Regarding the link, you should be able to download the document, here it is again:

https://drive.google.com/file/d/1Uei9VXd3Xk1inbgzKE7TF_xJ9ZOvqb6e/view?usp=sharing

And here is another implementation, this time its for a photography group who specialize in dental photography. They are all dentists:

https://ourworld.camera

Please tell me if there is any other way I can help you guys and fuel the "Buzz".......

10/10

Thank you
